src/ansys/mechanical/core/run.py

```diff
@@ -4,17 +4,19 @@
 from asyncio.subprocess import PIPE
 import os
 import sys
+import typing
 import warnings
 
 import ansys.tools.path as atp
 import click
 
+from ansys.mechanical.core.embedding.appdata import UniqueUserProfile
+
 # TODO - add logging options (reuse env var based logging initialization)
 # TODO - add timeout
-# TODO - close mechanical automatically after script exits
 
 
-async def _read_and_display(cmd):
+async def _read_and_display(cmd, env):
     """Read cmd's stdout, stderr while displaying them as they arrive."""
```

Comment:
Read command’s stdout and stderr and display them as they are processed.

Reply:
I'm not sure what "arrive" means in this instance.

Reply:
@sbradrick I think it means as the subprocess is running the command, the stdout and stderr messages are being displayed to the user as they are being returned in the subprocess. It allows you to see the subprocess output live instead of having to wait until the subprocess is done running, and then returning all of the stderr and stdout at once.

Reply:
> Read command’s stdout and stderr and display them as they are processed.

This description sounds good to me!